jobs/transforms/audits_registry.py

```diff
@@ -41,3 +42,25 @@ def get_audit_do_fn_types_by_kind():
         targeted.
     """
     return audit_decorators.AuditsExisting.get_audit_do_fn_types_by_kind()
+
+
+def get_id_property_targets_by_kind():
+    """Returns all registered ID properties and the models they target, keyed by
+    the kind of models the ID properties belong to.
+
+    Returns:
+        dict(str, dict(ModelProperty, tuple(str))). Model kinds mapped to the ID
+        properties they own and the kinds of models targeted by the ID property.
+    """
+    return audit_decorators.RelationshipsOf.get_id_property_targets_by_kind()
+
+
+def get_model_kinds_targeted_by_id_properties():
```

Comment:
get_all_model_kinds_referenced_by_id_properties (perhaps?)

I think the difficulty I have with "target" is that it implies a more active association than just storing the thing in a list.

Reply:
Ack'd, I think reference is a better word so I'll use it to replace target throughout.

Reply:
OK, how are these names?